src/telemetry/opentelemetry.ts

```diff
@@ -108,6 +108,13 @@ const instrumentations = [
 api.diag.setLogger(new api.DiagConsoleLogger(), api.DiagLogLevel.INFO);
 
 export const tracer = (serviceName: string) => {
+  if (process.env.OTEL_ENABLED !== 'true') {
```

Comment:
Just to make sure, is this going to be string from `otelEnabled` in yaml?

Reply:
Yeah